### PR TITLE
Startup remake

### PR DESCRIPTION
**Pull request by ssakone**

### CODE REVIEW

**Comment by ssakone** [comment]
I believe some Light theme color is not same as figma  @SylEze

**Comment by tonymorony** [comment]
1) this PR breaks login and wallet import screens (don't see such layout elements disappear on other branches):

Screenshot 2021-11-26 at 06 34 23
)

2) figma design functionality transferred not fully:

a) language switcher is missing - that's quite essential since the user should be able to use his native language during the wallet setup process, otherwise app translation loses its meaning

b) social network icons are missing

also, scroll looks different

3) overall impression is that there is not enough contrast between colors - at least on my displays (highlighted this place). @polevin could you please try this build https://github.com/KomodoPlatform/atomicDEX-Desktop/runs/4321837958?check_suite_focus=true to check it and advise alternative color? It is used in other places in application - can remember @SylEze raised problem with not enough contrast but not sure if consensus was reached (at least in figma it's still the same :) )

Screenshot 2021-11-26 at 06 34 57

**Review by tonymorony — CHANGES_REQUESTED** [review]
^^

**Comment by tonymorony** [comment]
good progress @ssakone , thank you!

0) seed confirmation process is broken

image

1) please move everything a bit down (except social network icons and lang selector)

image

2) please center this block both horizontally and vertically

Screenshot 2021-11-28 at 23 07 23

**Review by tonymorony — CHANGES_REQUESTED** [review]
^^

**Comment by tonymorony** [comment]
just found one more critical bug, I can't import wallet with custom seed (nothing happens when pressing "Next")

in logs:

```
[23:12:39] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:39] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:39] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:40] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:40] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:41] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:41] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:41] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:41] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:42] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:43] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:43] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
[23:12:43] [warning] [main.prerequisites.hpp:92] [838338]: qrc:/Dex/Screens/Startup/ImportWallet.qml:265: ReferenceError: tryPassLevel1 is not defined
```

**Comment by ssakone** [comment]
@tonymorony @smk762 @SylEze  Ready to review :)

**Comment by smk762** [comment]
red outline is a bit thicc on the sides..
image

If I confirm the seed, but then go back (from the set password screen), the red outline no longer appears on error
image

EULA / ToS button don't change visually when active
image


Some popups not matching figma
image
image
image


I see this in logs when scrolling thru my long list of wallets to select for login
`[10:11:10] [warning] [main.prerequisites.hpp:92] [948623]: qrc:/Dex/Screens/Startup/WalletsView.qml:132: TypeError: Cannot read property 'horizontalCenter' of null`

Logging in to new created wallet, I see in logs 
```
[10:12:19] [info] [qt.addressbook.page.cpp:66] [948623]: post_login: filling addressbook from cfg
[10:12:19] [warning] [main.prerequisites.hpp:92] [948623]: QIODevice::read (QFile, "/home/smk762/.atomic_qt/addressbook/qwerty"): device not open
[10:12:19] [warning] [addressbook.cfg.cpp:48] [948623]: Addressbook config file was invalid, use empty configuration: [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - unexpected end of input; expected '[', '{', or a literal. Content was: 
```

After creating wallet via import seed, it attempts autologin and gets stuck.

https://user-images.githubusercontent.com/35845239/143798809-1e0a27f6-de71-40ff-af51-5e5ca7246177.mp4


Language scrollbar slightly misaligned.
image

**Comment by ssakone** [comment]
@smk762  (After creating wallet via import seed, it attempts autologin and gets stuck.)

Its not something specific my PR break

**Comment by ssakone** [comment]
> Logging in to new created wallet, I see in logs

@SylEze  CC

**Comment by tonymorony** [comment]
`After creating wallet via import seed, it attempts autologin and gets stuck.` @smk762 could you please try to isolate this pb a bit? e.g. if it's the same on `design_tweaks` branch

**Comment by ssakone** [comment]
@smk762 @SylEze

**Comment by smk762** [comment]
Lang selector ok, but I dont see any red outline anymore on wrong seed word selection.

> EULA / ToS button don't change visually when active
Still see this.

>I see this in logs when scrolling thru my long list of wallets to select for login
> `[10:11:10] [warning] [main.prerequisites.hpp:92] [948623]: qrc:/Dex/Screens/Startup/WalletsView.qml:132: TypeError: Cannot read property 'horizontalCenter' of null`
Still get this.

Other stuff in my last comment I'll move to an issue.

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ include(vcpkg_prerequisites)
 include(qt_prerequisites)
 include(cfg_hash)
 
-project(${DEX_PROJECT_NAME} LANGUAGES CXX VERSION 0.5.3)
+project(${DEX_PROJECT_NAME} LANGUAGES CXX VERSION 0.5.2)
```
Comment (SylEze) [inline CMakeLists.txt:14]: Please revert this change

File: atomic_defi_design/Dex/Components/DexAppPasswordField.qml [hunk 2]
```diff
@@ -38,7 +38,7 @@ DexAppTextField
         width: 60
         radius: 20
         color: Dex.CurrentTheme.accentColor
-        border.color: _inputPassword.focus ? Dex.CurrentTheme.accentColor : DexTheme.rectangleBorderColor
+        border.color: _inputPassword.focus ? Dex.CurrentTheme.accentColor : color
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexAppPasswordField.qml:41]: Same color as background color, this statement is useless

File: atomic_defi_design/Dex/Components/DexComboBox.qml [hunk 3]
```diff
@@ -57,14 +57,14 @@ ComboBox
...
-        radius: 20
+        radius: 18
     }
 
     // Dropdown itself
     popup: Popup
     {
         width: control.width
-        height: _list.contentHeight > control.dropDownMaxHeight ? control.dropDownMaxHeight : _list.contentHeight
+        height: _list.contentHeight + 32 // > control.dropDownMaxHeight ? control.dropDownMaxHeight : _list.contentHeight
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexComboBox.qml:67]: Revert this change please, it will break every other combobox

File: atomic_defi_design/Dex/Components/DexComboBox.qml [hunk 4]
```diff
@@ -57,14 +57,14 @@ ComboBox
         implicitWidth: 120
         implicitHeight: 45
         color: control.mainBackgroundColor
-        radius: 20
+        radius: 18
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexComboBox.qml:60]: Revert this change it will break every other combobox

File: atomic_defi_design/Dex/Components/DexLanguage.qml [hunk 5]
```diff
@@ -54,4 +54,9 @@ DexComboBox {
             source: General.image_path + "lang/" + control.displayText + ".png"
         }
     }
+
+    DexMouseArea {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexLanguage.qml:58]: Use DefaultMouseArea and coding style

File: atomic_defi_design/Dex/Components/DexScrollBar.qml [hunk 6]
```diff
@@ -2,34 +2,36 @@ import QtQuick 2.15
 import QtQuick.Controls 2.15
 
 import App 1.0
+import Dex.Themes 1.0 as Dex
+
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexScrollBar.qml:6]: Useless newline

File: atomic_defi_design/Dex/Components/DexSwitch.qml [hunk 7]
```diff
@@ -75,4 +75,5 @@ Switch
         anchors.fill: parent
         acceptedButtons: Qt.NoButton
     }
+
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexSwitch.qml:78]: Useless newline

File: atomic_defi_design/Dex/Components/DexTooltip.qml [hunk 8]
```diff
@@ -1,8 +1,18 @@
 import QtQuick 2.15
 import QtQuick.Controls 2.15
+import QtQuick.Controls.Material 2.15
+import Dex.Themes 1.0 as Dex
+
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexTooltip.qml:5]: Useless newline

File: atomic_defi_design/Dex/Components/DexTooltip.qml [hunk 9]
```diff
@@ -1,8 +1,18 @@
...
+import QtQuick.Controls.Material 2.15
+import Dex.Themes 1.0 as Dex
+
 
 ToolTip
 {
-    id: root
+    id: control
+    
+    contentItem: DexLabel {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexTooltip.qml:11]: Coding style, use DefaultText

File: atomic_defi_design/Dex/Components/SetupPage.qml [hunk 10]
```diff
@@ -65,4 +66,12 @@ Item
             Layout.alignment: Qt.AlignHCenter
         }
     }
+
+    DexLanguage {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/SetupPage.qml:70]: Coding style

File: atomic_defi_design/Dex/Components/SquareButton.qml [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+
+import Qaterial 1.0 as Qaterial
+import Dex.Themes 1.0 as Dex
+
+Qaterial.SquareButton
+{   
+    elevation: 0
+    foregroundColor:  hovered ? Qt.lighter(Dex.CurrentTheme.foregroundColor) : Dex.CurrentTheme.foregroundColor
+
+    HoverHandler {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/SquareButton.qml:12]: Coding style

File: atomic_defi_design/Dex/Components/SquareButton.qml [hunk 12]
```diff
@@ -0,0 +1,15 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import Qaterial 1.0 as Qaterial
+import Dex.Themes 1.0 as Dex
+
+Qaterial.SquareButton
+{   
+    elevation: 0
+    foregroundColor:  hovered ? Qt.lighter(Dex.CurrentTheme.foregroundColor) : Dex.CurrentTheme.foregroundColor
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/SquareButton.qml:10]: Coding style

File: atomic_defi_design/Dex/Screens/Startup/Login.qml [hunk 13]
```diff
@@ -58,6 +58,11 @@ SetupPage
             topPadding: 10
         }
 
+        Item {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Screens/Startup/Login.qml:61]: Coding style

File: atomic_defi_design/Dex/Screens/Startup/Login.qml [hunk 14]
```diff
@@ -87,13 +92,16 @@ SetupPage
             visible: false
             field: _inputPassword.field
         }
+        Item {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Screens/Startup/Login.qml:95]: Coding style

File: atomic_defi_design/Dex/Screens/Startup/Login.qml [hunk 15]
```diff
@@ -107,15 +115,21 @@ SetupPage
             visible: false
         }
 
-        Qaterial.AppBarButton
+        DexAppButton
```
Comment (SylEze) [inline atomic_defi_design/Dex/Screens/Startup/Login.qml:118]: Use DefaultButton

File: atomic_defi_design/Dex/Screens/Startup/WalletsView.qml [hunk 16]
```diff
@@ -253,10 +256,23 @@ SetupPage
...
             }
+
+            
         }
+            
+        
         HorizontalLine { }
     }
 
+    LinksRow {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Screens/Startup/WalletsView.qml:267]: Coding style

File: atomic_defi_design/Dex/Settings/CamouflagePasswordModal.qml [hunk 17]
```diff
@@ -12,7 +12,7 @@ BasicModal {
     width: 800
 
     onClosed: {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Settings/CamouflagePasswordModal.qml:14]: Coding style

File: atomic_defi_design/Dex/Tests/ThemeTest.qml [hunk 18]
```diff
@@ -1,311 +1,38 @@
...
-	
 	id: root
-	property color backgroundColor: '#15182A' 
-	
-	Rectangle 
-	{
-		id: rect
-		anchors.fill: parent
-		color: Qt.rgba(.0, .0, .0,.11)
+	Column {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Tests/ThemeTest.qml:9]: Coding style

File: ci_tools_atomic_dex/installer/osx/packages/com.komodoplatform.atomicdex/meta/package.xml.in [hunk 19]
```diff
@@ -2,8 +2,8 @@
 <Package>
     <DisplayName>${DEX_DISPLAY_NAME}</DisplayName>
     <Description>Install ${DEX_DISPLAY_NAME}.</Description>
-    <Version>0.5.3</Version>
-    <ReleaseDate>2021-11-10</ReleaseDate>
+    <Version>0.5.2</Version>
```
Comment (SylEze) [inline ci_tools_atomic_dex/installer/osx/packages/com.komodoplatform.atomicdex/meta/package.xml.in:5]: Please update your version and regenerate files

**Review by SylEze — CHANGES_REQUESTED** [review]

File: atomic_defi_design/Dex/Components/DexComboBox.qml (hunk 4 above)
Comment (ssakone) [reply atomic_defi_design/Dex/Components/DexComboBox.qml:60]: All combobox in ui design kit have this radius, 
You think you can create a new combobox with different property?

**Comment by smk762** [comment]
> lgtm, @smk762 could you please also test - maybe I missed something

Same comments as previously.

> EULA / ToS button don't change visually when active

> I see this in logs when scrolling thru my long list of wallets to select for login
`[10:11:10] [warning] [main.prerequisites.hpp:92] [948623]: qrc:/Dex/Screens/Startup/WalletsView.qml:132: TypeError: Cannot read property 'horizontalCenter' of null`

Autologin on import also still an issue, but have moved to https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1461 to be dealt with in separate PR.

**Comment by ssakone** [comment]
@smk762 i doest not see any specific figma for eula model, but i make a new change according popup button layouting and style.

**Comment by tonymorony** [comment]
This PR contains many changes not related to it (assets configuration and etc), please open a clean one based on the latest `design_tweaks` branch with relevant changes only
